modified_cdc_protocol/operation_types/setup_pcr_plate/protocol.rb

```diff
@@ -0,0 +1,344 @@
+# typed: false
+# frozen_string_literal: true
+
+needs 'Standard Libs/PlanParams'
+needs 'Standard Libs/Debug'
+needs 'Standard Libs/Pipettors'
+needs 'Standard Libs/Units'
+needs 'Standard Libs/AssociationManagement'
+needs 'Standard Libs/LabwareNames'
+
+needs 'Collection Management/CollectionActions'
+
+needs 'Microtiter Plates/MicrotiterPlates'
+
+needs 'Microtiter Plates/PlateLayoutGenerator'
+
+needs 'PCR Libs/PCRComposition'
+
+needs 'Modified CDC Protocol/SetupPCRPlateDebug'
+
+needs 'Diagnostic RT-qPCR/DiagnosticRTqPCRHelper'
+
+PLATE = 'PCR Plate'
+LOT_NUM_KEY = 'Lot Number'
```

Comment:
Move to DataAssociationKeys?

Reply:
I was debating where that should live. I am starting to think it should live in DataAssociationKeys.